### ISSUE TITLE
remove quarries

### ISSUE THREAD

**Issue by bunniewormy** [transhumandesign/kag-base#328]
Quarries are extremely broken, even after the nerf. You can literally artificially make a stalemate all by yourself if you got the patience. This requires 0 skill, just boring minecrafting. I just finished a 1h55m match, which could go on for 1 hour more if I didn't decide to use 1k coins
![screen-18-12-25-16-06-40](https://user-images.githubusercontent.com/39147836/50425086-4e07bc80-0870-11e9-9c2d-6bf8ffa2dac7.png)
Setting stuff up, 15 minutes into the game
![screen-18-12-25-16-16-03](https://user-images.githubusercontent.com/39147836/50425091-59f37e80-0870-11e9-93d5-3370facc83b2.png)
Quarries set up already, 23m into the game. Defenses to the left were built by someone else using my stone, I didn't build anything yet
![screen-18-12-25-16-22-14](https://user-images.githubusercontent.com/39147836/50425096-80b1b500-0870-11e9-811a-ea385761527d.png)
30m into game, first storage got full so i needed to build another one
![screen-18-12-25-16-35-19](https://user-images.githubusercontent.com/39147836/50425097-8f986780-0870-11e9-8a64-8105cc38d3f1.png)
Started building at around 35m mark, this is a tower on 42m mark. Someone else made all these useless spikes - the stone wasted was produced back in maybe 5-10 min.
![screen-18-12-25-16-44-24](https://user-images.githubusercontent.com/39147836/50425100-a343ce00-0870-11e9-8600-8977a178e6fb.png)
53m
![screen-18-12-25-17-26-36](https://user-images.githubusercontent.com/39147836/50425104-b35bad80-0870-11e9-90e5-81720b0999f0.png)
1h30m
![screen-18-12-25-17-37-50](https://user-images.githubusercontent.com/39147836/50425111-c79faa80-0870-11e9-80ff-fa09a08bde8f.png)
1h45m
![screen-18-12-25-17-46-05](https://user-images.githubusercontent.com/39147836/50425114-d1291280-0870-11e9-872d-5ef0d88e34b9.png)
1h50m.
I just started spamming bomb arrows finally - had like 3k coins. The game could've lasted for much more if I didn't decide to win most likely.
In total I got probably over 10k coins, probably more. Died 4-6 times and disconnected once [had 4k coins at that point].

This isn't even a 'map problem' as many quarry fans tend to say. Keep in mind I did all of that almost all by myself, 2-3 other builders who used my stone did maybe 20% of what you see in the final screenshots. Count the stone doors as well please, 80% of them could've been wood but I didn't feel like wasting wood on that - using stone doors was literally more worth it with quarries... This could also easily be achievable with 3 quarries and less trees, even 2 quarries and 3-4 trees could do a lot. In fact, the game could probably be staled even more with less quarries, because I wouldn't build next towers as fast, but the enemies couldn't get past what I already built as well. It's just impossible with that amount of stone.

TL;DR:
1 builder can stale game for hours thanks to quarries, please remove, they were already nerfed and and it didn't help. Infinite stone just doesn't work in KAG.

**Comment by HappySheepo** [comment]
**_The problem is not the gold._**

In my opinion, i think there should be a maximum ammount of quarries built by both teams.

For example, if there's 1 quarry already on your team, you can't build any more quarries. With that in mind, gold will be used on upgrading bolts for ballies, tunnels, etc. And that would help to shorten the game.

With that limitation, minecrafting won't be as effective as it is now.

**Comment by bunniewormy** [comment]
closing since this is getting resolved in #democracy

**Comment by bunniewormy** [comment]
reopening - just did this again and it takes barely any effort. quarries should be limited to 1 per team or totally removed. destruction to building ratio was perfect before the addition of quarry, now games can get too stalematey because of this.
![screen-19-04-22-00-12-16](https://user-images.githubusercontent.com/39147836/56476496-fdf4db80-6498-11e9-8bba-0a5dcd9089f0.png)
![screen-19-04-22-00-13-30](https://user-images.githubusercontent.com/39147836/56476497-0220f900-6499-11e9-93a4-c8c51fce1a96.png)
![screen-19-04-22-00-28-07](https://user-images.githubusercontent.com/39147836/56476498-064d1680-6499-11e9-852a-afeada8e7b09.png)
probably should also remove the fact that you can get 500 wood with 100 coins by buying 5  crates and putting them into a saw
